dashboard/test/lib/services/script_seed_test.rb

```diff
@@ -531,6 +577,12 @@ def assert_resources_equal(resources1, resources2)
       end
     end
 
+    def assert_objectives_equal(objectives1, objectives2)
+      objectives1.zip(objectives2).each do |o1, o2|
+        assert_attributes_equal(o1, o2, ['lesson_id'])
```

Comment:
Ahh `lesson_id` is _excluded_ from comparison, not the property being compared on.

Reply:
yeah, I read this backwards the first time too! perhaps a keyword argument would help, but that can be for another time.